flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -237,12 +246,13 @@ public Task(TaskDeploymentDescriptor tdd,
 
 		this.memoryManager = checkNotNull(memManager);
 		this.ioManager = checkNotNull(ioManager);
-		this.broadcastVariableManager =checkNotNull(bcVarManager);
+		this.broadcastVariableManager = checkNotNull(bcVarManager);
+		this.accumulatorRegistry = accumulatorRegistry;
 
 		this.jobManager = checkNotNull(jobManagerActor);
 		this.taskManager = checkNotNull(taskManagerActor);
 		this.actorAskTimeout = new Timeout(checkNotNull(actorAskTimeout));
-		
```

Comment:
No auto formats, just removes redundant tabs in empty lines and corrects a missing space.